### PR TITLE
[OT185-19] News model and repository added

### PR DESCRIPTION
**Pull request by Sabrina-Incinga**
### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [ ] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

**Comment by Sabrina-Incinga** [comment]
Tuve que enviar de nuevo la pull request porque cometí el error de eliminar la otra rama al no saber cómo cambiar el nombre.

File: src/main/java/com/alkemy/ong/domain/model/News.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    @Column(name = "name", nullable = false, updatable = false)
+    private String name;
+
+    @Column(name="content", nullable = false, updatable = false)
+    private String content;
+
+    @Column(name = "image", nullable = false, updatable = false)
+    private String image;
+
+    @ManyToOne(cascade = CascadeType.MERGE)
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/News.java]: Por favor agregarle `fetch = FetchType.LAZY` y sacar `cascade = CascadeType.MERGE`

File: src/main/java/com/alkemy/ong/domain/model/News.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    @JoinColumn(name = "category_id", insertable = false, updatable = false)
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/News.java]: Cambiar **insertable** por **nullable**: `nullable = false`

File: src/main/java/com/alkemy/ong/domain/model/News.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    private Category category;
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/model/News.java:45]: Agregarle `@ToString.Exclude`

**Review by manerajona — CHANGES_REQUESTED** [review]

**Review by manerajona — APPROVED** [review]

**Pull request merged**